doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1,5 +1,5 @@
 FORMAT: 1A
-HOST: http://fiware-ngsi2.apiblueprint.org/
+HOST: http://orion.lab.fiware.org/
```

Comment:
Actually, what I would like to have it the operations rendered in the right panel as "GET /v2/entities" etc, without any host. However, I haven't been able (please tell me if you know how :) so I'm using the public Orion instance as reference (although currently it doesn't support v2, of course).